src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -214,7 +247,7 @@ public void resetHoodEncoder() {
      * @return The current angle of the hood
      */
     public double getHoodAngle() {
-        return hoodMotor.getSelectedSensorPosition() / DEGREES_TO_ENCODER_TICKS;
+        return hoodEncoder.getPosition();
     }
```

Comment:
Need to convert by the gear angle?  The PID reference below (which should match the encoder value) does that...

Reply:
In `configMotors()`, we set the conversion factor to `HOOD_DEGREES_TO_REVS`...
I didn't find a similar method in `SparkMaxPIDController` and didn't find one, but I should look into whether the encoder's conversion factor affects the PID.